src/main/java/org/fcrepo/spec/testsuite/authn/AuthenticatorResolver.java

```diff
@@ -61,7 +62,9 @@ public static void initialize(final String authenticatorClazz) {
     }
 
     private static Authenticator findAuthenticator() {
+        final PrintStream log = TestSuiteGlobals.logFile();
         try {
```

Comment:
Suggest changing `try` to: `try (final PrintStream log = TestSuiteGlobals.logFile()) {`
...and removing the `finally` block

Reply:
I'll change that.